ansible/roles/jetson_services/files/rover-base-station-gstreamer-0.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Base Station Gstreamer Service 0
+
+[Service]
+Environment="DISPLAY=:1"
```

Comment:
Not sure what the best way to do this with ansible is, but this is the variable we'd want to change based on what $DISPLAY really is

Reply:
Yeah this and the fact that ethernet ifaces are named differently on different laptops is going to poes a challenge.

Reply:
I think for DISPLAY we can just read the current environment variable for it. If you go in any shell and do `echo $DISPLAY` that *usually* gives you the right value.